string meas_type
time stamp
string src_asset
string measured_asset
float32 data
float32 variance
